src/client/components/pages/parts/editor-profile.js

```diff
@@ -188,12 +191,58 @@ class EditorProfileTab extends React.Component {
 		);
 	}
 
+	renderActivityGraph() {
+		const {activityData, totalRevisions} = this.props.editor;
+		const months = keys(activityData);
+		const numberOfRevisions = values(activityData);
+
+		if (!totalRevisions) {
+			return null;
+		}
+
+		const data = {
+			datasets: [
+				{
+					backgroundColor: 'rgba(235,116,59,0.2)',
+					borderColor: 'rgba(235,116,59,1)',
+					borderWidth: 1,
+					data: numberOfRevisions,
+					hoverBackgroundColor: 'rgba(235,116,59,0.4)',
+					hoverBorderColor: 'rgba(235,116,59,1)',
+					label: 'Revisions'
+				}
+			],
+			labels: months
+		};
+
+		return (
+			<div className="editor-activity-graph">
+				<div>
+					<h2>{this.props.editor.name}&apos;s Activity</h2>
+					<Line
+						data={data}
+						options={{
+							responsive: true
+						}}
+					/>
+				</div>
+			</div>
+		);
+	}
+
 	render() {
 		return (
 			<Row>
 				<Col md={12}>
 					{this.renderBasicInfo()}
+				</Col>
+				<Col md={12}>
 					{this.renderStats()}
+				</Col>
+				<Col md={9}>
```

Comment:
I think this size works fine.
I have a suggestion:  Making the stats column 3 wide and putting it after the graph. That way they look like a single row, which makes sense to me as they are related.
What do you think?

![FireShot Capture 045 - Monkey's Profile – BookBrainz - localhost](https://user-images.githubusercontent.com/6179856/78776669-8a452a80-7998-11ea-808c-178b575dc978.png)

Reply:
Or maybe have stats on the left and the graph to the right?

Reply:
![Screenshot_2020-04-08 prabalsingh24's Profile – BookBrainz(1)](https://user-images.githubusercontent.com/45737735/78777059-81635180-79b6-11ea-8959-d205ed05c4bc.png)

Reply:
I think this looks better. Without any heading 
![Screenshot_2020-04-08 prabalsingh24's Profile – BookBrainz(2)](https://user-images.githubusercontent.com/45737735/78777520-50cfe780-79b7-11ea-8991-d9894d6cd3a5.png)